Library/SharedFunctions.swift

```diff
@@ -778,16 +778,20 @@ private func estimatedMinMax(
   var max: Double = 0
 
   rewards.forEach { reward in
-    guard reward.shipping.enabled, let shippingRules = reward.shippingRules else { return }
+    guard reward.shipping.enabled, let shippingRules = reward.shippingRules else {
```

Comment:
Use `shippingRulesExpanded` here - this will have a shipping rule for each country. If `shippingRulesExpanded` isn't always available where we estimate shipping, we could use `shippingRules` as a backup for a "mostly working temp solution" but only if we're okay getting estimated shipping amounts wrong sometimes.